### PR TITLE
Add initial styling to self screener

### PR DESCRIPTION
**Pull request by devinjameson**
Why:
----
The self screener currently has no styling.

### CODE REVIEW

File: src/SelfScreener/CallEmergencyServices.tsx [hunk 1]
```diff
@@ -1,36 +1,122 @@
...
+      <View>
+        <SvgXml
+          xml={Icons.AsteriskInCircle}
+          fill={Colors.danger100}
+          width={Iconography.medium}
+          height={Iconography.medium}
+          style={style.icon}
+        />
+        <GlobalText style={style.headerText}>
+          {t("self_screener.call911.seek_medical_attention")}
```
Comment (johnschoeman) [inline src/SelfScreener/CallEmergencyServices.tsx:48]:
```suggestion
          {t("self_screener.call_emergency_services.seek_medical_attention")}
```

File: src/SelfScreener/SymptomCheckbox.tsx [hunk 2]
```diff
@@ -16,18 +18,63 @@ const SymptomCheckbox: FunctionComponent<SymptomCheckboxProps> = ({
...
   const checkboxIcon = checked ? Icons.CheckboxChecked : Icons.CheckboxUnchecked
+  const checkboxColor = checked ? Colors.primary100 : Colors.neutral75
+
+  const handleOnPressIn = () => {
+    setPressing(true)
+  }
+  const handleOnPressOut = () => {
+    setPressing(false)
+  }
+  const pressingStyle = pressing ? style.pressing : {}
```
Comment (johnschoeman) [inline src/SelfScreener/SymptomCheckbox.tsx:32]: fancy!

**Review by johnschoeman — APPROVED** [review]
LGTM, I might consider adding a before and after pic for context for code reviewers.

File: src/ExposureHistory/detail/ExposureActions.tsx [hunk 3]
```diff
@@ -118,8 +124,8 @@ const RequestCallBackActions: FunctionComponent<RequestCallBackActionsProps> = (
       </GlobalText>
       <Button
         customButtonStyle={style.requestCallbackButton}
-        customButtonInnerStyle={style.requestCallbackButtonGradient}
-        onPress={navigateToCallbackForm}
+        customButtonInnerStyle={style.requestCallbackButtonInner}
+        onPress={handleOnPressRequestCallback}
         label={t("exposure_history.exposure_detail.speak_with_contact_tracer")}
```
Comment (aledustet) [inline src/ExposureHistory/detail/ExposureActions.tsx:129]: Small 🥔  here but is the renaming to be consistent with what the button text is or to the action? If it is should we rename the translation to be request callback or the action itself?

File: src/SelfScreener/CallEmergencyServices.tsx (hunk 1 above)
Comment (aledustet) [reply src/SelfScreener/CallEmergencyServices.tsx:48]: Probably want to use the emergency number to populate the value of the translation also

**Review by aledustet — APPROVED** [review]
🚢  :shipit:

**Pull request merged**